utils/lists.ts

```diff
@@ -467,4 +467,9 @@ export const derivativesList: Record<string, string>[] = [
     description: "The first anti-metaverse built on Ethereum",
     url: "https://upsidedownloot.com/",
   },
+  {
+    name: "Loot Land",
```

Comment:
One last thing. Let’s put this entry sorted alphabetically by "name" field just like others. :)

Reply:
oh, no, ok. I'll update this.

Reply:
Maybe we need update the `package.json` to add some scripts for those check.

Reply:
done.